kratos/geometries/line_gl_3d_2.h

```diff
@@ -1,3 +1,4 @@
+#error is it used?
```

Comment:
Yes it is!

Reply:
OK, so I just remove the error. Meanwhile what is "gl" for?

Reply:
It means Gauss Lobatto. It would be nice to be able to define the geometry independently of the integration rule. Now a new object is needed for each integration rule.

Reply:
Your are right! As I remember the problem wasn't the Geometry itself (which can handle different integration type) but the GiDIO which cannot handle it. Isn't it? by the way  line_gauss_lobatto_... would be more descriptive....

Reply:
Ok I will change the name and I commit the change in this PR with the Point<3> to Point needed corrections from by side.

Reply:
Thank you @josep-m-carbonell! I will wait for your changes